aks/etcd-operator.yaml

```diff
@@ -0,0 +1,109 @@
+# concatenation of
```

Comment:
Why is this part of the aks code? I expected we would deploy the operator as part of the helm templates.

Reply:
I think operator per client is what we want. Other operators are going to be installed via helm charts/addons and will be per client so I don't think there is any reason to do something different for etcd. The only thing I expect we will get for free from the aks cluster is the etcdcluster CRD is already created (or is it created by the operator?).

Reply:
CRD is created by operator itself. Need to confirm this, but sound reasonable. I just followed the ingress controller and tiller model.

Reply:
> I think operator per client is what we want.

agree